openlibrary/solr/update_work.py

```diff
@@ -95,6 +95,7 @@ def set_solr_next(val: bool):
 class IALiteMetadata(TypedDict):
     boxid: set[str]
     collection: set[str]
+    access_restricted_item: str
```

Comment:
```suggestion
    access_restricted_item: Literal['true', 'false']
```

If this can be `None`, wrap it in `Optional[...]`